src/Console/Command.php

```diff
@@ -1,63 +1,10 @@
 <?php
 declare(strict_types=1);
 
-/**
- * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
- *
- * Licensed under The MIT License
- * For full copyright and license information, please see the LICENSE.txt
- * Redistributions of files must retain the above copyright notice.
- *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
- * @link          https://cakephp.org CakePHP(tm) Project
- * @since         3.6.0
- * @license       https://opensource.org/licenses/mit-license.php MIT License
- */
-namespace Cake\Console;
-
-use Cake\Datasource\ModelAwareTrait;
-use Cake\Log\LogTrait;
-use Cake\ORM\Locator\LocatorAwareTrait;
-
-/**
- * Base class for commands using the full stack
- * CakePHP Framework.
- *
- * Includes traits that integrate logging
- * and ORM models to console commands.
- */
-class Command extends BaseCommand
-{
-    use LocatorAwareTrait;
-    use LogTrait;
-    use ModelAwareTrait;
-
-    /**
-     * Constructor
-     *
-     * By default CakePHP will construct command objects when
-     * building the CommandCollection for your application.
-     */
-    public function __construct()
-    {
-        $this->modelFactory('Table', function ($alias) {
-            return $this->getTableLocator()->get($alias);
-        });
-
-        if (isset($this->modelClass)) {
-            $this->loadModel();
-        }
-    }
-
-    /**
-     * Implement this method with your command's logic.
-     *
-     * @param \Cake\Console\Arguments $args The command arguments.
-     * @param \Cake\Console\ConsoleIo $io The console io
-     * @return int|null|void The exit code or null for success
-     */
-    public function execute(Arguments $args, ConsoleIo $io)
-    {
-    }
-}
+class_alias(
+    'Cake\Command\Command',
+    'Cake\Console\Command'
+);
+deprecationWarning(
+    'Use Cake\Command\Command instead of Cake\Console\Command.'
```

Comment:
This is going to be a noisy deprecation. Can we leave this as a silent alias for the time being? This adds toil to upgrading with little value, as we learned with the ConsoleErrorHandler move uses get tripped up by these.

Reply:
Yup I was thinking that too. Will remove the warning.

Reply:
> Should I move Command class under Command namespace as @othercorey suggested and add class alias for it's existing location?

The `Command` class being in the package is only a problem if people try to use it. Could we solve that problem with documentation instead of adding deprecation warnings that impact applications and plugins for a standalone package that may not have widespread adoption.

Reply:
So, as a new user, I followed the documentation on the README as that was the first and only thing I saw.  There is no mention of the Command class there.  However, if you click on the links a the bottom for other things, you can find the Command Objects documentation which starts with explaining using the Command class.

It's awkward it exists but isn't usable, but this is just FYI.  I'm not making an argument either way.